apps/remix-ide/src/blockchain/blockchain.tsx

```diff
@@ -889,8 +904,13 @@ export class Blockchain extends Plugin {
       let execResult
       let returnValue = null
       if (isVM) {
-        const hhlogs = await this.web3().remix.getHHLogsForTx(txResult.transactionHash)
+        if (!tx.useCall) {
```

Comment:
Saving the state should be done elsewhere, so it also takes in account txs done from a script.